File: android/src/com/flagquest/game/AndroidLauncher.java
package com.flagquest.game;

import android.os.Bundle;

import com.badlogic.gdx.backends.android.AndroidApplication;
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration;
// import com.flagquest.game.FlagQuest;

public class AndroidLauncher extends AndroidApplication {
	@Override
	protected void onCreate (Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		AndroidApplicationConfiguration config = new AndroidApplicationConfiguration();
		initialize(new FlagQuest(), config);
	}
}

File: client/android/src/com/flagquest/game/AndroidLauncher.kt
package com.flagquest.game

import android.os.Bundle
import com.badlogic.gdx.backends.android.AndroidApplication
import com.badlogic.gdx.backends.android.AndroidApplicationConfiguration
import com.flagquest.game.models.AuthHandler
import com.google.firebase.FirebaseApp
import com.google.firebase.auth.FirebaseAuth


class AndroidLauncher : AndroidApplication() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        FirebaseApp.initializeApp(this)
        val config = AndroidApplicationConfiguration()
        AuthHandler.initialize(FirebaseAuthHandler())
        initialize(FlagQuest(), config)
    }
}
